samples/database.sample.yml

```diff
@@ -0,0 +1,670 @@
+# Mappings to SQLLite Database
+#
+# FST lemma             = lemma.cw
```

Comment:
This suggests that we are only referring to CW content, or does it refer to CW-style orthography (i.e. using the accented <y>)?

My understanding has been that our crk-db will have aggregate entries, which have their own head (label), which is based on some normatized form of the heads of the entries in the source dictionaries. Typically, that will be the CW form, but we will have some entries that are only to be found in MD, or AECD, so we will then have to create the normatized form used as the head in crk-db ourselves.

Reply:
It refers to a CW-style orthography with the <ý>. @eddieantonio suggested we use `sro` for the plain <y> orthography and `cw` for the orthography with Arok's <ý>, and I think `sro` is the right choice for the plain <y> variety, but I can see the confusion in using `cw` for the other one. What if I we called it the "historical orthography" (`hist`) or "diachronic orthography" (`dia`), since that <ý> represents a historic phoneme. Or maybe "pandialectal orthography" (`pan`) or "multidialectal orthography" (`multi`)? Those would all be more accurate descriptions of that orthography.

I'll use `multi` for now.

Reply:
Another possibility: `proto` for protolanguage.

Reply:
`proto` for Proto-Cree! See also: https://en.wikipedia.org/wiki/Cree_language#Dialect_criteria